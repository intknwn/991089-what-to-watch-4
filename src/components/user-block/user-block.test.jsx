import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import UserBlock from '../user-block/user-block.jsx';

const user = {
  id: 1,
  email: `Oliver.conner@gmail.com`,
  name: `Oliver.conner`,
  avatarSrc: `/img/1.png`,
};

it(`UserBlock component should render correctly`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <UserBlock
          isAuthorized={true}
          user={user}
        />
      </BrowserRouter>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
