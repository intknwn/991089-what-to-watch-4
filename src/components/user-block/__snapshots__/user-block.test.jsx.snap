// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserBlock component should render correctly 1`] = `
<div
  className="user-block"
>
  <div
    className="user-block__avatar"
  >
    <a
      href="/mylist"
      onClick={[Function]}
    >
      <img
        alt="Oliver.conner@gmail.com"
        height={63}
        src="https://4.react.pages.academy/img/1.png"
        width={63}
      />
    </a>
  </div>
</div>
`;
